lib/utils.js

```diff
@@ -15,6 +15,24 @@ function Utils(config) {
      */
     var utils = {};
 
+    utils.decode = function(buf, callback) {
+        var feature;
+        try {
+            feature = geobuf.geobufToFeature(buf);
+        } catch (err) {
+            if (callback) return callback(err);
+            else throw err;
+        }
+
+        for (var key in feature.properties) {
+            try { feature.properties[key] = JSON.parse(feature.properties[key]); }
+            catch (err) { /* no-op */ }
```

Comment:
This PR is an attempt to explore a non-breaking change to cardboard that can solve this problem, but I am beginning to think that there's no such thing. You're right about "42" and there are a number of other reasons that this is probably not a good long-term solution.